README.md

```diff
@@ -24,10 +24,16 @@ A tool for content designers to present complex information as a flow of questio
 
 This is a Ruby on Rails app, and should follow [our Rails app conventions](https://docs.publishing.service.gov.uk/manual/conventions-for-rails-applications.html).
 
-You can use the [GOV.UK Docker environment](https://github.com/alphagov/govuk-docker) or the local `startup.sh` script to run the app. Read the [guidance on local frontend development](https://docs.publishing.service.gov.uk/manual/local-frontend-development.html) to find out more about each approach, before you get started.
+You can use the [GOV.UK Docker environment](https://github.com/alphagov/govuk-docker) or the local `startup.sh --live` script to run the app. Read the [guidance on local frontend development](https://docs.publishing.service.gov.uk/manual/local-frontend-development.html) to find out more about each approach, before you get started.
 
 If you are using GOV.UK Docker, remember to combine it with the commands that follow. See the [GOV.UK Docker usage instructions](https://github.com/alphagov/govuk-docker#usage) for examples.
 
+You can run the tests with one of the rake tasks as listed by `rake --tasks`
+
+### Troubleshooting
+When running integration tests, if you get `SessionNotCreatedException: Message: session not created: This version of ChromeDriver only supports Chrome version` error, then run `brew install chromedriver` or if you already have the cask, `brew upgrade chromedriver`
```

Comment:
I find that installing and running chromedriver this way doesn't work for me, so I always just install the executable into `/usr/local/bin`. Some of the other GOV.UK devs have had similar experiences and may have adopted the same approach. Might want to offer this as an alternative, if the brew way doesn't work.